tests/mongo-tools/dump_restore_test.go

```diff
@@ -21,57 +21,77 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	"go.mongodb.org/mongo-driver/bson"
 
 	"github.com/FerretDB/dance/tests/common"
 )
 
 func TestDumpRestore(t *testing.T) {
 	ctx, db := common.Setup(t)
 
-	// TODO restart existing dump from https://github.com/mcampo2/mongodb-sample-databases instead
-	_, err := db.Collection("test").InsertOne(ctx, bson.D{{"foo", "bar"}})
+	localActualPath := filepath.Join("..", "..", "dumps", "actual")
+	containerActualPath := "/dumps/"
+
+	localExpectedPath := filepath.Join("..", "..", "dumps", "expected")
+	containerExpectedPath := "/sample-dumps/"
+
+	dbName := "sample_geospatial"
```

Comment:
So, in the scope of the current task, we use only this dump. Are there plans to test more dumps in the future, or do we consider this one enough?

Reply:
We needed this dump to test some large real-word dataset. I believe, we don't have any plans for other test cases

Reply:
Ok!